leveldb/Log/json/KVJson.cpp

```diff
@@ -0,0 +1,82 @@
+// warm remind: gcc KVJson.cpp -lstdc++ -o KVJson
+// difficult with GCC: https://en.wikibooks.org/wiki/JsonCpp
+
+// Use another single file
+
+#include <iostream>
+#include <chrono>
+#include <ctime> 
+#include <typeinfo>       // operator typeid
+#include <string>     // to use std::string, std::to_string() and "+" operator acting on strings 
+
+#include "json.hpp"
+//#include <nlohmann/json.hpp>
+
```

Comment:
Since this is about KVJson, I think creating a .h file to define all member variables is useful.